DashAI/back/models/parameters/models_schemas/RandomForestClassifier.json

```
@@ -61,8 +61,8 @@
         "random_state": {
             "oneOf": [
                 {
-                    "error_msg": "El parámetro 'random_state' debe ser un entero mayor o igual a 0.",
-                    "description": "El parámetro 'random_state' debe ser un entero mayor o igual a 0.",
+                    "error_msg": "The 'random_state' parameter must be an integer greater than or equal to 0.",
+                    "description": "The 'random_state' parameter must be an integer greater than or equal to 0.",
                     "type": "integer",
```

Same as before. We should specify that the parameter is nullable

```suggestion
                    "error_msg": "The 'random_state' parameter must be an integer greater than or equal to 0.",
                    "description": "Controls both the randomness. Should be an integer.",
                    "type": ["integer", "null"],
```